### ISSUE TITLE
What is 5th Index value in Precision, Recall and F1 lists

### ISSUE THREAD

**Issue by vjkadekar** [xashru/punctuation-restoration#4]
Hi, 
I got the below values for one of the tests. 

Precision: [0.98068136 0.71008241 0.87272477 0.83011424 0.81440205]
Recall: [0.98558426 0.67705532 0.84840781 0.84434041 0.79418293]
F1 score: [0.9831267  0.69317568 0.86039451 0.83716689 0.80416542]
Accuracy:0.9535017031737457
Confusion Matrix
[[297677   2467   1236    651]
 [  3773  12839   1806    545]
 [  1366   2295  25364    871]
 [   725    480    657  10100]]

On manually calculating, first four indexes of Precision. Recall and F1 score correspond to 4 punctuations tried. But what does 5th one refer to? Its definitely not Average of the first 4 to represent "Overall" value.

**Comment by xashru** [comment]
This is indeed the overall score but ignoring the results for no punctuation or 'O' entries in labels.
You can check this in line #181 in train.py

**Comment by vjkadekar** [comment]
> This is indeed the overall score but ignoring the results for no punctuation or 'O' entries in labels.
> You can check this in line #181 in `train.py`

Ok.. thank you. Also, can you let me know after how many epochs, the algorithm converged? I searched for this on the paper but couldnt find any info on it.

**Comment by xashru** [comment]
We trained for 10 epochs and used the model with the best validation accuracy (evaluated after each epoch). I don't have exact numbers, and it varied among models, but most of the time it converged around 7-8 epochs.

**Comment by vjkadekar** [comment]
Thanks much...
